### ISSUE TITLE
(ISO 6709) PRF stage document cover page and title representation different

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-iso#684]
Left: Metanorma-generated. Right: Original
<img width="1538" alt="Screenshot 2022-05-20 at 6 25 58 PM" src="https://user-images.githubusercontent.com/11865/169508917-c868d7e6-05d1-4aa8-80f7-4f88d2598529.png">

<img width="1648" alt="Screenshot 2022-05-20 at 6 28 33 PM" src="https://user-images.githubusercontent.com/11865/169509350-031d8947-04b8-4b7f-8110-2f81d1543465.png">

Metadata:
```adoc
= ISO 6709:2022(E)
:docnumber: 6709
:ref-docnumber: ISO 6709:2022(E)
:doctype: international-standard
:copyright-year: 2022
:published-date: 2022-03-03
:language: en
:edition: 3
:docstage: 60
:docsubstage: 60
:title-main-en: Standard representation of geographic point location by coordinates
:title-main-fr: Représentation normalisée de la localisation des points géographiques par coordonnées
:technical-committee-number: 211
:technical-committee: Geographic information/Geomatics
:workgroup-type: WG
:workgroup-number: 9
:secretariat: SIS
:ics: 35.240.70
:mn-document-class: iso
:imagesdir: images
:mn-output-extensions: xml,html,doc,pdf,html_alt,rxl
:local-cache-only:
:data-uri-image:
:revises: ISO 6709:2008;ISO 6709:2008/Cor 1:2009
```

**Comment by opoudjis** [comment]
Also, need to do away with zzAddress style, since it is resulting in spurious boxes.

**Comment by opoudjis** [comment]
DIS styles are currently only operational for styles 40 and 50, and are not activated for either 60.00 or 60.60. I have explicitly told you they are not yet activated for published documents. This ticket is forcing me to queue-jump that, and will therefore only be actioned after all other ISO 6709 tickets are done, when I will enable DIS styles for published documents.

**Comment by opoudjis** [comment]
The zzAddress contents are unindented in ISO DIS 19115-3, indented in ISO 6709, double spaced in both. Need to move out of zzAddress formatting. If indent is required, will insert manual formatting in postprocessing, but it seems that manual formatting is done only on publication.

**Comment by opoudjis** [comment]
zzAddress rendering addressed in https://github.com/metanorma/metanorma-iso/issues/653

**Comment by opoudjis** [comment]
Will leave background colours on for stages 40 and 50, will turn off colours in CSS for PRF.

**Comment by ronaldtse** [comment]
I'm not sure how this is to be verified. This is a PRF document.

Left: Metanorma at stage 50.60 (FDIS). Right: ISO at stage 60.00 (which is PRF).

<img width="1290" alt="Screenshot 2022-05-21 at 6 49 10 PM" src="https://user-images.githubusercontent.com/11865/169648225-832c5389-1cc9-41ba-a20d-fcd3d5517778.png">

1. At PRF stage, we need to enable 60.00 to show a PRF style cover
2. Date should be 2nd field in style zzCover, not Body Text
3. At PRF stage, reference number is just "ISO 6709:2022(E)" without the label
4. At PRF stage, the label "Committee identification:" is to removed.
5. French title should be italicized.
6. Reference number (1st line), Date (2nd line), and English title (5th line) should be bolded.
7. The zzAddress is breaking the zzCopyright box into 2. In addition, there is an extra "white border" box that holds zzCopyright content together.

**Comment by opoudjis** [comment]
I warned you, very explicitly, about bringing up

> There is no "COPYRIGHT PROTECTED DOCUMENT" text in the Edited DIS template.

https://github.com/metanorma/metanorma-iso/issues/653#issuecomment-1133550238

> I will note that the published samples you sent me DO NOT include the COPYRIGHT PROTECTED DOCUMENT title, as you can see, but I am at this stage so sick of the ongoing clusterfuck lack of clarity around DIS rendering, that I'm just doing what you told me.

You INSISTED that I put the COPYRIGHT PROTECTED DOCUMENT back in. You are now insisting that I take it back out.

I am not going to touch it. You are failing to get your act together, and wasting my time.

**Comment by opoudjis** [comment]
Given the lack of concrete guidance, I am yet again being forced to guess.

There now are at least three, possibly four distinct title page formats, which I am being forced to infer by telepathy.

* So there is the original title page, which emulated that of published documents.
* There is the title page for DIS, for stages 40 and 50
* As you have just told me now, there is a distinct PRF title page. The fact that PRF is using a distinct coverpage form that of published documents is completely insane, but then, so is ISO editorial in general

So what I need to do now is:

* create a distinct PRF front page, which reflects the difference between the ISO 6709 ed.3 document (PRF) and the ISO_DIS 19115-3 ed.1 document (DIS)
* apply these changes to them
* Differentiate address indenting between DIS and PRF, because somehow that is a feature and not a bug
* Remove the COPYRIGHT PROTECTED DOCUMENT in DIS, but put it back in for PRF, but only for PDF output
* And I'm going to do that by stripping the title for Word output, until I'm given an actionable alternative; creating a placeholder is not an actionable alternative, because something has to go into it.

**Comment by ronaldtse** [comment]
There is no need to guess. I will provide samples here to show what exactly is happening. As you know the PDF and Word layouts are different, and that also applies to the PRF stage.

**Comment by ronaldtse** [comment]
Apologies that I have made some mistakes in the comparison of the documents. The ISO editorial process is highly erratic and clearly that different editors generate slightly different outputs. I failed to properly generalize and describe several issues that I will attempt to rectify here.

> Differentiate address indenting between DIS and PRF, because somehow that is a feature and not a bug

No, I don't think address non-indent is a feature. It's likely a bug with ISO editors.

In the 60.60 stage, the "zzCopyright" text is entirely gone. Otherwise in all other stages, they are present, identically.

> Remove the COPYRIGHT PROTECTED DOCUMENT in DIS, but put it back in for PRF, but only for PDF output

* All PDF outputs have this "COPYRIGHT PROTECTED DOCUMENT".
* All Word outputs do not have "COPYRIGHT PROTECTED DOCUMENT".

From 690 DIS:
<img width="574" alt="Screenshot 2022-05-23 at 11 14 53 AM" src="https://user-images.githubusercontent.com/11865/169736398-92b87f1c-76e5-44cb-ba4a-5112e546e55c.png">
